Purchases/Misc/Deprecations.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  Deprecations.swift
+//
+//  Created by Nacho Soto on 3/8/22.
+
+import Foundation
+import StoreKit
+
+// swiftlint:disable line_length missing_docs
```

Comment:
I've created this similarly to `Obsoletions.swift` where we can put all these.
I think the `swiftlint:disable` helps, we could have probably done the same in `Obsoletions.swift`. Easier to read with long lines instead of breaking every platform, and we don't actually want to document these. What do you all think?

Reply:
yeah, good call 👍